third_party/blink/renderer/modules/ml/model.cpp

```diff
@@ -174,6 +192,36 @@ void Model::addOperation(int32_t type,
       ml::mojom::blink::Operation::New(type, inputs, outputs));
 }
 
+void Model::setOperandSymmPerChannelQuantParams(
+    uint32_t index,
+    const OperandSymmPerChannelQuantParam* operandSymmPerChannelQuantParam,
+    ExceptionState& exception_state) {
+  if (InvalidAction(is_finished_, exception_state))
+    return;
+
+  ml::mojom::blink::OperandPtr& operand = model_info_->operands[index];
+  if (operandSymmPerChannelQuantParam->channelDim() < 0 ||
+      operandSymmPerChannelQuantParam->channelDim() >=
+          operand->dimensions.size())
+    return;
```

Comment:
It seems to return silently. Would you like to indicate this is an invalid parameter?

Reply:
I add a new method `InvalidOperandSymmPerChannelQuantParams` to indicate the invalid message.